new/LIT7107Divination.xml

```diff
@@ -0,0 +1,155 @@
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2024-10-21">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <note>The following incipits are taken from <ref type="mss" corresp="BAVet119"/>.</note>
+                
+                <div type="textpart" xml:id="table1">
+                    <label>Table divided into sixteen columns and seven orders</label>
+                    <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <ab>
+                            ፩<supplied reason="omitted">፡</supplied> አላህያን፡ በዕል፡ ዕለት። በመጋቢት፡ በ ፳፱<supplied reason="omitted">፡</supplied>
+                            <choice><sic>ይሠርቀ</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ይሠርቅ፡</corr></choice> ፳፫<supplied reason="omitted">፡</supplied> ያበርህ፡ ዕለት፡ 
+                            <surplus>ዕለት፡</surplus> ቤተ፡ ፀሩ፡ አልኬድ፡ ፪፭፮፯፲ ወ ፩፲ ወ፪<supplied reason="omitted">፡</supplied> ቤተ፡ ክፍሉ<supplied reason="omitted">፡</supplied>
+                            ፩፫፬፰፱፲ወ፬፲ወ፭ወ፲ወ፮<supplied reason="omitted">፡</supplied> ኮከቡ<supplied reason="omitted">፡</supplied> እልመሸተሪ፡ ዕለ<supplied reason="omitted">ቱ፡
+                                ሐ</supplied> <supplied reason="omitted">ሙስ፡</supplied>
+                        </ab>
+                    </div>                    
+                </div>
+                
+                <div type="textpart" xml:id="table2">
+                    <label>Table containing the names of the sixteen constellations of the previous table and some directives on their use</label>   
+                    <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <ab>
+                            ለፋል፡ ዘይትገደፍ፡ ለ<add place="above">ነ</add>ቢር<supplied reason="omitted">፡</supplied> ስም፡ ወእም፡ ወምድር<supplied reason="omitted">፡</supplied> 
+                            ለኮከብኒ፡ ስም፡ ወእም<supplied reason="omitted">፡</supplied> ለገይስ፡ ስም፡ ወእም<supplied reason="omitted">፡</supplied> ወርኅ፡ ዕለት፡ በ፲ወ፮ ግድፍ፡
+                        </ab>
+                    </div>
+                </div>
+                
+                <div type="textpart" xml:id="chapters">
+                    <label>Sixteen chapters</label>
+                   <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <ab>
+                            ፩ ቀዳማዊ፡ ዐዐዐዐዐዐዐ አላዘያን፡ ዘወጽአ፡ ለሠናይ፡ ዝንቱ፡ ቤት፡ ቤተ፡ ነፍስ፡ ወሕይወት፡ ወዝንቱ፡ ትእምርት፡ ኅሩይ፡ <sic>እሳቲዊ፡</sic> ወከዋክብት፡ አልመሻታሪ፡ 
+                            <sic>ወእመዋዕል፡</sic> ሐሙስ፡ ወእግዚአብሔር፡ <sic>የአምር፡</sic> በአሰንዮ፡ ንብረት፡ ወኵሉ፡ ፍጥረት፡ ርቱዕ፡ ወብዙኅ፡ <sic>ፍሥሐ፡</sic> ወሐሤት፡ ወፍጻሜ፡ 
+                            መፍቅድ፡ <sic>ወጸሕቅ፡</sic> <choice><sic>ወአግርር፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወአግርሮ፡</corr></choice>
+                            ፀላእት፡ <choice><sic>ወነጸሮ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወነጽሮ፡</corr></choice> ሕልም፡ ወአስተጋብዖ፡ ሠናያት፡ እምለፌ፡ ወእምለፌ፡
```

Comment:
ፀላእት፤ in the manuscript. If you standardised the punctuation, maybe it would be possible to state that where you say where the text has been taken from?